### PR TITLE
Describe Sync Lock with Dynamic Tempo

### PR DESCRIPTION
**Pull request by daschuer**
https://deploy-preview-589--mixxx-manual.netlify.app/chapters/djing_with_mixxx#sync-lock-with-dynamic-tempo

### CODE REVIEW

File: source/chapters/djing_with_mixxx.rst [hunk 1]
```diff
@@ -159,6 +159,62 @@ properly.
...
+   :width: 124px
+   :alt: Sync Leader button as soft leader
+
+.. |explicitleader| image:: ../_static/2.4/explicit-leader.png
+   :width: 124px
+   :alt: Sync Leader button as explicit leader
+
+You can create a variable BPM beat grids for single tracks by using the :guilabel:`Analyze`
+library context menu and :guilabel:`Reanalyze (variable BPM)` command. Alternatively you may
+create a variable BPM beat grid for all tracks by unchecking :guilabel:`Assume constant tempo`
```
Comment (ronso0) [inline source/chapters/djing_with_mixxx.rst:188]:
```suggestion
create variable BPM beat grids for all tracks to be (re)analyzed later on by unchecking :guilabel:`Assume constant tempo`
```

File: source/chapters/djing_with_mixxx.rst [hunk 2]
```diff
@@ -159,6 +159,62 @@ properly.
...
+in the :guilabel:`Beat Detection` preferences.
+
+When such a track is playing in a Leader deck, the follower decks closely follows the tempo.
```
Comment (ronso0) [inline source/chapters/djing_with_mixxx.rst:191]:
```suggestion
When such a track is playing in a Leader deck, the follower decks closely follow the tempo.
```

File: source/chapters/djing_with_mixxx.rst [hunk 3]
```diff
@@ -159,6 +159,62 @@ properly.
...
+It's important to note that the track follows the "detected" leader tempo unconditionally,
+even if there are sections with unstable tempos during bridges or in the outro regions where
+the actual played tempo isn't correctly detected.
```
Comment (ronso0) [inline source/chapters/djing_with_mixxx.rst:194]:
```suggestion
the actual played tempo wasn't detected correctly.
```

File: source/chapters/djing_with_mixxx.rst [hunk 4]
```diff
@@ -159,6 +159,62 @@ properly.
...
+
+If a soft leader deck stops or is silenced, Mixxx automatically selects the next best
+deck as Leader. By clicking the Leader button, you can toggle between Explicit Leader
+and Soft Leader mode. An Explicit leader is sticky, does not automatically move.
+
+By default the "Sync Lock with Dynamic Tempo" feature is enabled and Mixxx picks the
+first Sync Lock enabled track as Soft Leader, using the tempo from its outro during
+the following transition. If you'd like to use the the tempo from the intro of the next
+track instedad, click the Leader (Crown) button of its deck.
```
Comment (ronso0) [inline source/chapters/djing_with_mixxx.rst:203]:
```suggestion
track instead, click the Leader (Crown) button of its deck.
```

File: source/chapters/djing_with_mixxx.rst [hunk 5]
```diff
@@ -159,6 +159,62 @@ properly.
...
+The deck becomes than the explicit leader. By another click on the Leader button the
```
Comment (ronso0) [inline source/chapters/djing_with_mixxx.rst:204]:
```suggestion
The deck then becomes the explicit leader. Click again on the Leader button and the
```

File: source/chapters/djing_with_mixxx.rst [hunk 6]
```diff
@@ -159,6 +159,62 @@ properly.
...
+deck becomes Soft-follower and Mixxx is again in charge to move the leader to
+the best deck.
+
+If you'd prefer a steady tempo during transitions like in Mixxx before Version 2.4,
+you may select :guilabel:`Use steady Tempo` for :guilabel:`Sync mode` in the
+:guilabel:`Decks` preferences. In this mode all decks become a follower during a
+transition and you may pick a dynamic track as a leader by cklick the
```
Comment (ronso0) [inline source/chapters/djing_with_mixxx.rst:211]:
```suggestion
transition and you may pick a dynamic track as a leader by clicking the
```

File: source/chapters/djing_with_mixxx.rst [hunk 7]
```diff
@@ -159,6 +159,62 @@ properly.
...
+Leader (Crown) button if desired.
+
+A leader plays always the original recorded tempo changes. A follower change its
```
Comment (ronso0) [inline source/chapters/djing_with_mixxx.rst:214]:
```suggestion
A leader plays always the original recorded tempo changes. A follower changes its
```

**Review by ronso0 — CHANGES_REQUESTED** [review]
Just some typos/rephrasing

**Comment by daschuer** [comment]
Thank you.

**Comment by daschuer** [comment]
OK 2.3 is merge to 2.4, translations are updated and this is rebased on top of all.

**Comment by ronso0** [comment]
Cool.
I just tried to enhance the text flow. Even though I think the text is comprehensible I doubt I have a 'naive look' at the text, i.e. put myself into a noob's position.
Someone else should double-check before we merge this.

**Comment by daschuer** [comment]
@ywwg Can you have a look?

**Review by ywwg — APPROVED** [review]
no notes! Well-described and written.  Thank you.

**Comment by ronso0** [comment]
Thanks for your review!

**Pull request merged**